03.asynchronous/callback_success.js

```diff
@@ -0,0 +1,19 @@
+import sqlite3 from "sqlite3";
+
+const db = new sqlite3.Database(":memory:");
+
+db.run(
+  "CREATE TABLE books(id INTEGER PRIMARY KEY AUTOINCREMENT, title TEXT NOT NULL UNIQUE)",
+  () =>
+    db.run(
+      "INSERT INTO books(title) VALUES(?)",
+      "JSふりがなプログラミング",
+      function () {
+        console.log(`id: ${this.lastID}`);
+        db.get("SELECT * FROM books", (_, book) => {
+          console.log(`id: ${book.id}, title: ${book.title}`);
+          db.run("DROP TABLE books", () => db.close());
```

Comment:
ここは return したいわけではないので、簡潔文体ではなくブロック文体を使ったほうが分かりやすいんじゃないでしょうか。簡潔文体を使っても実害はないのですが、ブロック文体で書いたほうが戻り値を使う意図がないことを明確に表せます。

```suggestion
          db.run("DROP TABLE books", () => {
            db.close()
          });
```

Reply:
ありがとうございます！
「戻り値を使うかどうか」を基準としてアロー関数の文体を見直しました～